src/Microsoft.SqlTools.ServiceLayer/EditData/Session.cs

```diff
@@ -21,13 +23,9 @@ namespace Microsoft.SqlTools.ServiceLayer.EditData
     public class Session
```

Comment:
I know this wasn't changed in this PR, but EditDataSession might be a better name for this class. It's in the EditData namespace, but still.